### PR TITLE
[IMP] barcord: Inventory Adjustment

### PR DESCRIPTION
**Pull request by anr-odoo**
shows a kanban view of the draft and in progress inventory adjustment, I click on it, it opens the barcode view of the inventory adjustment and  add a create button to start an inventory adjustment from scratch to provide more efficiency and convenience to user.

Task : https://www.odoo.com/web#id=1970451&action=327&model=project.task&view_type=form&menu_id=4720

Pad : https://pad.odoo.com/p/r.14bd692e7a6b5301c19b90bc044729b3

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/views/stock_inventory_views.xml [hunk 1]
```diff
@@ -136,7 +136,9 @@
                             </div>
                             <div class="o_kanban_record_bottom">
                                 <div class="oe_kanban_bottom_left">
-                                    <i class="fa fa-clock-o" role="img" aria-label="Date" title="Date"/> <t t-esc="record.date.value"/>
+                                </div>
+                                <div class="oe_kanban_bottom_right">
+                                    <t t-esc="record.date.value and record.date.value.split(' ')[0] or False"/>
```
Comment (sle-odoo) [inline addons/stock/views/stock_inventory_views.xml]: is it the best way to dispay a datetime in date on a kanban? are they no widgets?

Comment (hsh-odoo) [reply addons/stock/views/stock_inventory_views.xml]: @sle-odoo Done

File: addons/stock/views/stock_inventory_views.xml [hunk 2]
```diff
@@ -149,7 +149,9 @@
                             </div>
                             <div class="o_kanban_record_bottom">
                                 <div class="oe_kanban_bottom_left">
-                                    <i class="fa fa-clock-o" role="img" aria-label="Date" title="Date"/> <t t-esc="record.date.value"/>
+                                </div>
+                                <div class="oe_kanban_bottom_right">
+                                    <field name="date" widget="date"/>
```
Comment (sle-odoo) [inline addons/stock/views/stock_inventory_views.xml]: the commit message seems completely out of sync with the content?

File: addons/web/static/src/js/chrome/action_manager_act_window.js [hunk 3]
```diff
@@ -677,6 +677,10 @@ ActionManager.include({
                 }
                 ctx.add(action.context || {});
                 action.context = ctx;
+                var res_id = ctx.eval().res_id;
+                if (res_id) {
+                    action.res_id = res_id;
+                }
```
Comment (sle-odoo) [inline addons/web/static/src/js/chrome/action_manager_act_window.js]: why do you need this commit? what's the usecase? do you need it in our branch?

Comment (sle-odoo) [reply addons/web/static/src/js/chrome/action_manager_act_window.js]: @pgu-odoo

**Comment by sle-odoo** [comment]
robodoo r+

**Pull request closed without merging**